release-calendar/src/client/components/Calendar.tsx

```diff
@@ -29,6 +30,9 @@ export class Calendar extends React.Component<CalendarProps, {}> {
   render(): JSX.Element {
     return (
       <div className='calendar'>
+        <div className='title-bar'>
+          <h1>Calendar Title</h1>
+        </div>
```

Comment:
Why is this added?

Reply:
It helps to horizontally align everything. "Calendar Title" is just a placeholder. I can add a comment about that to clarify.

Reply:
The calendar is pretty self explanatory though. Maybe you can play around with the channel table title.